avionics/common/include/radio.h

```diff
@@ -62,6 +53,29 @@ class Radio {
     // Loose type to more explicitly indicate what variables are commands.
     typedef uint8_t command_t;
 
+    enum Ids : command_t {
```

Comment:
I still think you should delete the typedef 2 lines above and just have this one be like:
```suggestion
    enum command_t : uint8_t {
```
I just don't think there's a reason to have two types representing the same thing. Unless you had a reason?

Reply:
My bad. I was getting some errors while building that I thought were unresolvable with this change but it turns out they were due to an unrelated issue.